draft-irtf-cfrg-hpke.md

```diff
@@ -713,13 +730,15 @@ see {{hpke-export}} for more details.
 It is up to the application to ensure that encryptions and
 decryptions are done in the proper sequence, so that encryption
 and decryption nonces align. If a Seal or Open operation would cause the `seq`
-field to overflow, then the implementation MUST return an error. Note that
+field to overflow, then the implementation MUST return an error. (In the
+pseudocode below, `IncrementSeq` fails with an error when `seq` overflows,
+which causes `Context.Seal` and `Context.Open` to fail accordingly.) Note that
```

Comment:
Suggestion, to make it clearer about which Seal and Open this is, and maybe failing with an error is more explicitly saying that no other value can be returned (like, not an error *and* the ciphertext): “If `Context.Seal` or `Context.Open` would cause the `seq` field to overflow, then the implementation MUST fail with an error.”

Reply:
That works for me -- I'll add it. Thanks!